lib/backends/jcr/tree.js

```diff
@@ -57,6 +57,8 @@ var TEMP_FILE_PATTERNS = [
   // Windows
   /^desktop\.ini/i,
   /^Thumbs\.db/i,
+  /Adobe\ Bridge\ Cache.bc/,
+  /Adobe\ Bridge\ Cache.bct/,
```

Comment:
In order to make the patterns more restrictive to this case, I might recommend changing them to:

`/^Adobe\ Bridge\ Cache\.bc$/i`
`/^Adobe\ Bridge\ Cache\.bct$/i`

This restricts the pattern to the full file name (so that only these files will match), ignores case, and escapes the dot in the file name so that it will match a literal dot instead of any character.